packages/db/src/schema.prisma

```diff
@@ -286,6 +292,30 @@ model Gloss {
   @@id([wordId, languageId])
 }
 
+/// Represents a gloss update for a particular language.
+model GlossHistoryEntry {
+  /// The word whose gloss is being updated.
+  word       Word        @relation(fields: [wordId], references: [id])
+  /// The ID of the word whose gloss is being updated.
+  wordId     String
+  /// The language of the updated gloss.
+  language   Language    @relation(fields: [languageId], references: [id])
+  /// The ID of the language of the updated gloss.
+  languageId String      @db.Uuid
+  /// The timestamp when the gloss was updated.
+  timestamp  DateTime    @default(now())
+  /// The user who updated the gloss.
+  user       AuthUser?   @relation(fields: [userId], references: [id])
+  /// The ID of the user who updated the gloss
+  userId     String?     @db.Uuid
+  /// The updated text of the gloss. Null indicates no change.
+  gloss      String?
+  /// The updated approval state of the gloss. Null indicate no change.
+  state      GlossState?
```

Comment:
Let's add a field called source that takes a new enum `GlossSource` that has two variants `USER` and `IMPORT`. That way we can distinguish between whether this history entry came from an import or not

Reply:
Done.